google-client-loader.html

```diff
@@ -162,19 +162,22 @@
 
       _fireSuccess: function() {
         this.fire(this.successEventName,
-            { 'name': this.name, 'version': this.version });
+            { 'name': this.name, 'version': this.version },
+            { composed: true });
```

Comment:
Actually, the API on `fire` has changed since this was written; it sets `composed` to true by default now. I will just revert the line as it is unnecessary